src/assets/i18n/es.json

```
@@ -241,12 +292,21 @@
     "total_plan_ad": "Total Plan de Adquisiciones: ",
     "asignar_ingreso": "Asignar Ingreso",
     "cambiar_ingreso": "Cambiar Ingreso",
-    "saldo_restante_mov": "Saldo restante del movimiento"
+    "saldo_restante_mov": "Saldo restante del movimiento",
+    "fuentes_asignadas": "Fuentes Asignadas"
   },
   "ACTIVIDAD":{
     "comprobacion-diferencia": "El valor de la(s) actividad(es) supera el de la apropiación por un valor de: ",
     "codigo": "Código",
-    "seleccionar_actividad": "Recuerde seleccionar una actividad en la lista desplegable"
+    "seleccionar_actividad": "Recuerde seleccionar una actividad en la lista desplegable",
+    "actividad_creada": "Actividad Creada",
+    "actividad": "Actividad",
+    "actividades": "Actividades",
+    "valor_actividad_error": "Es necesario el valor de la actividad",
+    "actividades_asociadas": "Actividades Asociadas",
+    "todas_actividades_asociadas": "Todas las actividades disponibles están asociadas",
+    "agregar_actividad": "Agregar Actividad",
+    "": "Editar Actividad"
```

REVISAR: Quizás faltó esto...? (Revisar antes de aplicar este _suggested change_)
```suggestion
    "editar_actividad": "Editar Actividad"
```